deegree-core/deegree-connectionprovider-datasource/src/main/java/org/deegree/db/datasource/DataSourceConnectionProvider.java

```diff
@@ -118,12 +118,16 @@ public Connection getConnection() {
     @Override
     public void destroy() {
         if ( destroyMethod != null ) {
+            LOG.info("Closing connection pool " + resourceMetadata.getIdentifier());
             try {
                 destroyMethod.invoke( ds );
             } catch ( Exception e ) {
                 String msg = "Error destroying DataSource instance: " + e.getLocalizedMessage();
                 LOG.error( msg );
             }
+        } else {
+            LOG.warn("Unable to close connection pool " + resourceMetadata.getIdentifier()
+                    + ". Check the DataSource configuration if the attribute 'destroyMethod' is configured." );
```

Comment:
We would appreciate it if string placeholders were used, as this would result in a more clean code.